src/core.js

```diff
@@ -106,10 +106,35 @@ $.extend($.fn, {
 		}
 		return valid;
 	},
+	validAsync: function(callback) {
+		var validator, interval, valid = this.valid();
+
+		if ( $( this[ 0 ] ).is( "form" ) ) {
+			validator = this.validate();
+		} else {
+			validator = $( this[ 0 ].form ).validate();
+		}
+
+		if ( typeof(callback) !== "function" ) {
```

Comment:
Adding a separate method seems okay, but it doesn't make sense to have an optional callback then. Let's drop this check and return.